Solutions/Syslog/Hunting Queries/squid_volume_anomalies.yaml

```diff
@@ -1,6 +1,8 @@
 id: e472c490-4792-4f12-8b6b-6ab3e0404d35
 name: Squid data volume timeseries anomalies
 description: |
+    'Malware or data exfiltration can cause network data volume anomalies. This query detects such anomalies in bytes traversing a squid proxy, requiring further investigation. Assumes default squid log format'
```

Comment:
'Malware or data exfiltration can cause network data volume anomalies. This query detects such anomalies in volume of bytes traversing a squid proxy. Anomalies require further investigation. Assumes default squid log format.'

Reply:
Resolved